core/src/main/java/com/orange/ouds/core/component/OudsNavigationBar.kt

```diff
@@ -0,0 +1,511 @@
+                        strokeWidth = 1.dp.toPx()
+                    )
+                },
+            containerColor = if (translucent && Build.VERSION.SDK_INT > Build.VERSION_CODES.S_V2) colorBgTranslucent.value else colorBgOpaque.value,
+            contentColor = colorContentUnselectedEnabled.value,
+            windowInsets = windowInsets,
+            content = content
+        )
+    }
+}
+
+/**
+ * An OUDS navigation bar item.
+ *
+ * It must be used in [OudsNavigationBar].
+ *
+ * The recommended configuration for an [OudsNavigationBarItem] depends on how many items there are inside a [OudsNavigationBarItem]:
+ * - Three destinations: Display icons and text labels for all destinations.
+ * - Four destinations: Active destinations display an icon and text label. Inactive destinations display icons, and text labels are recommended.
+ * - Five destinations: Active destinations display an icon and text label. Inactive destinations use icons, and use text labels if space permits.
+ *
+ * A [OudsNavigationBarItem] always shows text labels (if it exists) when selected. Showing text labels if not selected is controlled by [alwaysShowLabel].
+ *
+ * @param selected Whether this item is selected or not.
+ * @param onClick Called when this item is clicked.
+ * @param icon Icon of the item.
+ * @param modifier [Modifier] applied to the navigation bar item.
+ * @param label Label of the item.
+ * @param badge Optional badge display on the item icon.
+ * @param alwaysShowLabel Whether the label should always be shown.
+ * @param interactionSource [MutableInteractionSource] that will be used to dispatch events when this item is pressed, hovered or focused.
+ *
+ * @sample com.orange.ouds.core.component.samples.OudsNavigationBarSample
+ */
+@Composable
+fun RowScope.OudsNavigationBarItem(
+    selected: Boolean,
+    onClick: () -> Unit,
+    icon: OudsNavigationBarItemIcon,
+    modifier: Modifier = Modifier,
+    label: String? = null,
+    badge: OudsNavigationBarItemBadge? = null,
+    alwaysShowLabel: Boolean = true,
+    interactionSource: MutableInteractionSource? = null
+) {
+    @Suppress("NAME_SHADOWING") val interactionSource = interactionSource ?: remember { MutableInteractionSource() }
+    val interactionState by interactionSource.collectInteractionStateAsState()
+    val state = getNavigationBarItemState(interactionState = interactionState)
+    with(OudsTheme.componentsTokens.bar) {
+        val selectedContentColor = contentColor(state = state, selected = true)
+        val unselectedContentColor = contentColor(state = state, selected = false)
+
+        ConstraintLayout(
+            modifier = modifier
+                .selectable(
+                    selected = selected,
+                    onClick = { },
+                    role = Role.Tab
+                )
+        ) {
+            val (itemRef, topIndicatorRef) = createRefs()
+
+            // Top active indicator: visual alternative for selected item (a11y)
```

Comment:
This was a comment from ODS, is a11y the actual reason why we have these indicators in OUDS?